Bottom: ed58a354edeb7dd8b911c0f31c86466d5f750f5b
Top:    35d5c157f5c994e1483f67deea52fb711f3c974a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-13 01:53:43 +0100

Refresh of fix-some-broken-assumptions

---

diff --git a/src/polygon.c b/src/polygon.c
index bd6352a..013b6aa 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -303,6 +303,8 @@ original_poly (PolygonType * p)
           poly_InclVertex (contour->head.prev, poly_CreateNode (v));
         }
 
+      frac_circle (..);
+
       /* Is current point last in contour? If so process it. */
       if (n == p->PointN - 1 ||
           (hole < p->HoleIndexN && n == p->HoleIndex[hole] - 1))
diff --git a/src/polygon1.c b/src/polygon1.c
index 1cf24db..4598f3c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2904,8 +2904,7 @@ poly_InsideContour (PLINE * c, Vector p)
   info.f = 0;
   info.p[0] = ray.X1 = p[0];
   info.p[1] = ray.Y1 = p[1];
-#warning BROKEN ASSUMPTION of 32 BIT
-  ray.X2 = 0x7fffffff;
+  ray.X2 = COORD_MAX;
   ray.Y2 = p[1] + 1;
   if (setjmp (info.env) == 0)
     r_search (c->tree, &ray, NULL, crossing, &info);
